lib/prepare_business_uk_leaving_eu.yaml

```diff
@@ -117,10 +117,15 @@ pages:
       <p>Doing business abroad includes exporting and going for meetings or taking goods to a trade fair.</p>
   employ_eu_citizens:
     show_in_qa: true
-    question_type: "single"
+    question_type: "single_custom_options"
     question: "Do you employ anyone from another European country?"
     description: |
-      <p>This includes <a href="/eu-eea">countries that are in the EU</a> and Norway, Switzerland, Iceland and Liechtenstein.</p>
+      <p>This includes <a href="/eu-eea">countries in the EU</a> and Norway, Switzerland, Iceland and Liechtenstein.</p>
+    custom_options:
+      - label: "Yes"
+        value: "yes"
```

Comment:
Can you a note in the commit message saying that this only works if the values match the values in the facet_group:  https://github.com/alphagov/content-tagger/blob/master/lib/data/find-eu-exit-guidance-business.yml#L243

Reply:
I see you've updated the commit message which is great, but I think it needs to be clear that it's the facet_values config (published by content tagger) that needs to match, not the finder content_item (published by search-api).

> It is important to note that each value must match a facet value
in the finder which it will redirect to, otherwise, it would not be
applied and ignored by the finder. These lists are not validated against
the finder, therefore, if the finder facet values change, this will need
to be changed too.